### ISSUE TITLE
CTM and Chisel Mod Update breaks Faithful32 connected textures for some blocks

### ISSUE THREAD

**Issue by blumn2** [F32Organization/Faithful32-1.12.2#138]
Recently, updates for the Chisel and CTM mods were released. I am not an expert at modding, but they must have changed the way the file structure was organized, because now when I have Faithful32 enabled, it does not recognize the connected textures of multiple blocks, including multiple variants of stone bricks. 

With Faithful32 enabled:
![image](https://user-images.githubusercontent.com/45048997/68551239-d3d51d80-03d8-11ea-8013-614440ed9eca.png)

With Faithful32 disabled:
![image](https://user-images.githubusercontent.com/45048997/68551262-10a11480-03d9-11ea-930d-f459493fbf83.png)

Like I said earlier, I'm no expert at modding, so I am not sure what exactly changed on Chisel/CTM's part. If there is anything I can try and do to help please let me know. 

Thank you all for all you do with this Resource Pack!

**Comment by blumn2** [comment]
Update: So, after posting this issue, I looked into it and it appears that Chisel and CTM have officially dropped support for the .ctx files. I forked the repo and have started to replace the outdated .ctx files with .mcmeta files - A process that took a lot of trial and error. I will continue to replace the .crx files with the .mcmeta files and then maybe I can request a pull request? I haven't used GitHub like this before so I am still figuring it all out.

tl;dr: .ctx files used by this modpack are no longer supported, I have been going through and replacing them with the .mcmeta equivalents to fix the issue.

Thanks again!

**Comment by blumn2** [comment]
I have finished replacing the .ctx files with .mcmeta files, and made a pull request. I think that's what I needed to do? 

Thanks again, and let me know if I did this right!
